src/test/java/com/michelin/ns4kafka/integration/ConnectorIntegrationTest.java

```diff
@@ -648,5 +650,7 @@ private void waitForConnectorAndTasksToBeInState(String connector, Connector.Tas
                 // Connector not found, retry
             }
         }
+
+        Thread.sleep(2000);
```

Comment:
Is this necessary?

Reply:
It helped a bit when there were issues in integration tests getting rebalanced. I will definitely remove it